WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -746,7 +769,7 @@ def __startJobAgent(self):
 
     retCode, _output = self.executeAndGetOutput( jobAgent, self.pp.installEnv )
     if retCode:
-      self.log.error( "Could not start the JobAgent" )
+      self.log.error( "Error executing the JobAgent [ERROR %d]" % retCode )
```

Comment:
No, because if we want to have something really generic, in the pilots, we shouldn't even assume that DIRAC is installed. That's why there's no "import DIRAC" anywhere in pilots code. I'll code something generic enough for the pilots.